clients/lsp-eslint.el

```diff
@@ -346,7 +389,7 @@ to allow or deny it.")
                                    (funcall callback))
                                (error (funcall error-callback err))))
                            error-callback
-                           :url (lsp-vscode-extension-url "dbaeumer" "vscode-eslint" "2.1.14")
+                           :url (lsp-vscode-extension-url "dbaeumer" "vscode-eslint" "2.1.23")
```

Comment:
I wish we do that change in a way to force downloading of the new version in case it is already downloaded the previous one.

Reply:
I'm not 100% sure what the best way to do this is - to my knowledge, the version isn't stored anywhere so it's tricky to know if the server needs to be updated. Any suggestions?

Reply:
I think that slightly changing the download location should work. We might include the server version in the new store path path. `lsp-eslint-unzipped-path` controls that.

Reply:
Added the version number to `lsp-eslint-unzipped-path`, that should do it for most cases. If the user is setting it to a custom value though, the update won't be forced. There's very little reason for users to be changing this anyway though - do we think this is much of a problem?

Reply:
I think this PR is the best we can do. Can you also drop a note in the changelog? After that, it is good to go.